pomegranate/base.pyx

```diff
@@ -137,6 +138,29 @@ cdef class Model(object):
 		"""
 		raise NotImplementedError
 
+	def plot( self, n=1000, **kwargs ):
```

Comment:
Adding a plot function should be a separate PR. I would prefer not to merge several disparate features in a single PR unless necessary.